sources/packages/backend/apps/api/src/institution-integration/ier-integration/ier12-file-detail.ts

```diff
@@ -0,0 +1,79 @@
+import { DisbursementSchedule, DisbursementValue } from "@sims/sims-db";
+import {
+  DATE_FORMAT,
+  NUMBER_FILLER,
+  SPACE_FILLER,
+} from "../../cra-integration/cra-integration.models";
+import { round, StringBuilder } from "../../utilities";
+import { IER12FileLine } from "./models/ier12-integration.model";
+
+/**
+ * Record of a IER12 request file.
+ * The documentation about it is available on the document
+ * 'SIMSSFAS - Institution File layouts In Analysis Folder'.
+ */
+export class IER12FileDetail implements IER12FileLine {
+  assessmentId: number;
+  applicationNumber: string;
+  sin: string;
+  studentLastName: string;
+  studentGivenName: string;
+  birthDate: string;
+  programName: string;
+  programDescription: string;
+  credentialType: string;
+  cipCode: number;
+  nocCode: string;
+  sabcCode: string;
+  institutionProgramCode: string;
+  programLength: number;
+  studyStartDate: string;
+  studyEndDate: string;
+  tuitionFees: number;
+  programRelatedCosts: number;
+  mandatoryFees: number;
+  exceptionExpenses: number;
+  totalFundedWeeks: number;
+  disbursementSchedules: DisbursementSchedule[];
+
+  getFixedFormat(): string {
+    const disbursementValuesTotal: DisbursementValue[] =
+      this.disbursementSchedules.flatMap(
+        (disbursement) => disbursement.disbursementValues,
+      );
+    const awardsSum: Record<string, number> = {};
```

Comment:
Thanks for changing the calculation but is the `awardsSum` being added to the file content?